module/data/constants.py

```diff
@@ -2,7 +2,10 @@
 
 START_CMD_TEXT = "Benvenuto! Questo bot è stato realizzato dagli studenti del Corso di Laurea in Informatica"
 
-HELP_CMD_TEXT = """📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio"""
+HELP_CMD_TEXT = '\n'.join((
+    "📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio",
+    "🔑 /register Permette di effettuare la registrazione al sistema"
+))
```

Comment:
Puoi usare una stringa piu' lunga con le triple vigolette o il backslash, non c'e' bisogno di creare una tupla e fare il join.

Reply:
Mi sembrava meglio farla cosi' per un discorso di leggibilita' del file in se, pero' se avete fatto sempre con le triple quotes mi adeguo

Reply:
Qua ci sono alcune opzioni per ottenere lo stesso risultato. Vedi quali preferisci.

```py
ORIGINAL = "\n".join(
    (
        "📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio",
        "🔑 /register Permette di effettuare la registrazione al sistema",
    )
)

BACKSLASH = "📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio\n\
🔑 /register Permette di effettuare la registrazione al sistema"

TRIPLE_QUOTE = """📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio
🔑 /register Permette di effettuare la registrazione al sistema"""

CONCATENING = (
    "📬 /report Fornisce la possibilità di poter inviare una segnalazione agli sviluppatori riguardante qualsiasi disservizio\n"
    "🔑 /register Permette di effettuare la registrazione al sistema"
)
```